frontend/dialogs/OBSWhatsNew.cpp

```diff
@@ -1,17 +1,17 @@
-#include "moc_window-whats-new.cpp"
+#include "OBSWhatsNew.hpp"
 
-#include <QPushButton>
-#include <QHBoxLayout>
-#include <QVBoxLayout>
-
-#include "window-basic-main.hpp"
+#include <widgets/OBSBasic.hpp>
 
 #ifdef BROWSER_AVAILABLE
 #include <browser-panel.hpp>
 extern QCef *cef;
 #endif
 
-/* ------------------------------------------------------------------------- */
+#include <QPushButton>
+#include <QHBoxLayout>
+#include <QVBoxLayout>
```

Comment:
```suggestion
#include <QHBoxLayout>
#include <QPushButton>
#include <QVBoxLayout>
```

If alphabetical is preferred, order like this.